x-pack/plugins/actions/server/actions_client/actions_client.ts

```diff
@@ -255,6 +256,33 @@ export class ActionsClient {
     }
     this.context.actionTypeRegistry.ensureActionTypeEnabled(actionTypeId);
 
+    const hookServices: HookServices = {
+      scopedClusterClient: this.context.scopedClusterClient.asCurrentUser,
```

Comment:
For my specific use case I need `scopedClusterClient.asInternalUser`, so can we just return `scopedClusterClient` to have a choice or expose two client?